lib/utils/plugins.js

```diff
@@ -130,7 +121,26 @@ const plugins = (config, craDeps) => {
     if (!hmrCompiler.supportsHMR) throw new BrunchError('HMR_PLUGIN_UNSUPPORTED');
   }
 
-  return getIncludes(plugins).then(includes => {
+  /* Get paths to files that plugins include. E.g. handlebars-brunch includes
+   * `../vendor/handlebars-runtime.js` with path relative to plugin.
+   */
+  const getIncludes = () => {
```

Comment:
Maybe it would be better to take `plugins` as an argument and not to use it from global scope.